test/driver.ts

```diff
@@ -32,6 +32,7 @@ export type DriverOptions = {
     readyToSyncTimeout: number;
     banningThreshold: number;
     web3Provider : () => Web3;
+    contractRegistryAddress: string | undefined;
```

Comment:
Better to define the property as optional using `?`.
`contractRegistryAddress?: string`